inductiva/resources/machines_base.py

```diff
@@ -142,10 +143,61 @@ def from_api_response(cls, resp: dict):
 
         return machine_group
 
-    def start(self, **kwargs):
+    def _update_termination_timers(self,
```

Comment:
Agree that this should be exposed to the user. It can be helpful if, for example the user, underestimated the time some simulation would take